homepage.html

```diff
@@ -0,0 +1,271 @@
+                class="form-control"
+                id="address"
+                placeholder="Waterloo"
+              />
+            </div>
+            <div class="form-group">
+              <label for="exampleInputPassword1">
+                Number of bed
+              </label>
+              <input
+                type="text"
+                class="form-control"
+                id="beds"
+                placeholder="1"
+                onchange="remap()"
+              />
+            </div>
+            <div class="form-group">
+              <label for="exampleInputPassword1">
+                Check in
+              </label>
+              <input type="date" class="form-control" id="date1" />
+            </div>
+            <div class="form-group">
+              <label for="exampleInputPassword1">
+                Check out
+              </label>
+              <input type="date" class="form-control" id="date2" />
+            </div>
+            <button type="submit" class="btn btn-primary">
+              Search
+            </button>
+          </form>
+        </div>
+        <div class="col-md-6">
+          <div class="show" id="show">
+            <span id="sentence">Explore new airbnb:</span>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div id="map"></div>
+    <script>
+      var map;
+      var lat;
+      var lng;
+      function initMap() {
+        map = new google.maps.Map(document.getElementById("map"), {
+          center: { lat: -34.397, lng: 150.644 },
+          zoom: 8
+        });
+      }
+      function remap() {
+        var addr = document.getElementById("address").value;
+        var geocoder = new google.maps.Geocoder();
+        geocodeAddress(geocoder, map);
+        function geocodeAddress(geocoder, resultsMap) {
+          geocoder.geocode({ address: addr }, function(results, status) {
+            if (status === "OK") {
+              resultsMap.setCenter(results[0].geometry.location);
+              map.setZoom(20);
+              // var marker = new google.maps.Marker({
+              //   map: resultsMap,
+              //   position: results[0].geometry.location
+              // });
+              lat = results[0].geometry.location.lat();
+              lng = results[0].geometry.location.lng();
+            } else {
+              alert(
+                "Geocode was not successful for the following reason: " + status
+              );
+            }
+          });
+        }
+      }
+      var search_form = document.getElementById("search-form");
+
+      search_form.onsubmit = function(e) {
+        e.preventDefault();
+        document.getElementById("show").innerHTML = "";
+        var title1 = document.createElement("span");
+        title1.id = "sentence";
+        title1.innerHTML = "Search result:";
+        $(".show").append(title1);
+        var addr = document.getElementById("address").value;
+        var begindate = document.getElementById("date1").value;
```

Comment:
please follow the convention of naming of JavaScript.

this link may be useful: [variable naming principles](https://www.w3cplus.com/javascript/variable-naming-principles.html)